fedimint-core/src/module/mod.rs

```diff
@@ -578,6 +580,26 @@ pub trait CommonModuleInit: Debug + Sized {
     fn decoder() -> Decoder;
 }
 
+/// Can be used to send a notification to consensus server,
+/// to trigger immediate [`ServerModule::consensus_proposal`] call.
+///
+/// Useful for modules to optionally decrease latency of consensus
+/// item handling.
+#[derive(Debug, Clone)]
+pub struct ConsensusProposalTrigger(watch::Sender<u64>);
```

Comment:
@maan2003 didn't you build something similar recently? I think it was called `Wait`. Could we maybe unify the underlying mechanism? I like this approach.

Reply:
The mechanism is unified already. It's `tokio::sync::watch`. :D

Here I'm just wrapping it in a newtype, so the API is more 🌈*type-driven*🌈.

Reply:
I misremembered, the other abstraction ([`Waiter`](https://docs.fedimint.org/fedimint_core/task/waiter/struct.Waiter.html)) had slightly different semantics so probably not worth switching to this approach.